applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -439,8 +455,21 @@ static void le_audio_msg_sub_thread(void)
 			LOG_INF("Presentation delay %d us is set by initiator", pres_delay_us);
 			break;
 
+		case LE_AUDIO_EVT_SYNC_LOST:
+			if (IS_ENABLED(CONFIG_BT_OBSERVER)) {
```

Comment:
Should this have been
```suggestion
			if (IS_ENABLED(CONFIG_BT_BAP_BROADCAST_SINK)) {
```

Reply:
Or `CONFIG_TRANSPORT_BIS` as used further below?

Reply:
I'm trying to move away from the more general `CONFIG_TRANSPORT_BIS`, so will change it to `CONFIG_BT_BAP_BROADCAST_SINK`